src/mock-data/houses.json

```diff
@@ -35,7 +37,9 @@
     "rating": 4.4,
     "title": "Large house for the whole family",
     "location": "Los Angeles",
-    "description": "Lorem ipsum is placeholder text commonly used in the graphic, print, and publishing industries for previewing layouts and visual mockups."
+    "description": "Lorem ipsum is placeholder text commonly used in the graphic, print, and publishing industries for previewing layouts and visual mockups.",
+    "checkInDate": "Thursday, 18 May 2022 06:02:17",
+    "checkOutDate": "Tuesday, 03 Jun 2023 17:36:20"
```

Comment:
I thought this should be dynamic =)

Reply:
https://github.com/igorizviekov/mt-booking_app/pull/43/commits/d183357d595863576e00f1ec5c65383ed5ea589d